knowledge-base/kb-datetimevalues-to-formattedstrings.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Formatting Display of DateTime Values in Selection Box
```

Comment:
Add the name of the control in the `title`, `page_title` and `description`, for better visibility.